presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/BasePlanTest.java

```diff
@@ -92,6 +93,11 @@ public final void destroyPlanTest()
         queryRunner = null;
     }
 
+    public ConnectorId getCurrentConnectorId()
```

Comment:
Use it in `com.facebook.presto.sql.planner.sanity.TestValidateAggregationsWithDefaultValues#setup` too